CHANGELOG.rst

```diff
@@ -4,6 +4,9 @@ Changelog
 =========
 
 
+- Add ``lower_case``.
```

Comment:
Go ahead and add `Thanks bharadwajyarlagadda_!` after the changelog entry (see other examples in the file). And at the bottom of the page add:

```rst
.. _bharadwajyarlagadda: https://github.com/bharadwajyarlagadda
```

Gotta give credit where credit is due :+1: